### PR TITLE
Revert "duplicate asset ID" and apply "get_txids_filtered" interface.

### PR DESCRIPTION
**Pull request by libscott**
Revert "Merge pull request #1043 from bigchaindb/duplicate-asset-id"

This reverts commit 8e497347d52b7c65aef180a88df9a8f615ef6736, reversing
changes made to a706d88b303f748b1a6e01b884761de62e2e35cb.

### CODE REVIEW

File: bigchaindb/backend/mongodb/query.py [hunk 1]
```diff
@@ -83,6 +85,43 @@ def get_blocks_status_from_transaction(conn, transaction_id):
 
 
 @register_query(MongoDBConnection)
+def get_txids_filtered(conn, asset_id, operation=None):
+    parts = []
+
+    if operation != Transaction.TRANSFER:
```
Comment (sbellem) [inline bigchaindb/backend/mongodb/query.py]: Out of curiosity: Why not simply:

```python
if operation == Transaction.CREATE:
```

File: bigchaindb/backend/mongodb/query.py [hunk 2]
```diff
@@ -83,6 +85,43 @@ def get_blocks_status_from_transaction(conn, transaction_id):
...
+            {'$unwind': '$block.transactions'},
+            {'$match': {
+                'block.transactions.id': asset_id,
+                'block.transactions.operation': 'CREATE'
+            }},
+            {'$project': {'block.transactions.id': True}}
+        ])
+        parts.append(elem['block']['transactions']['id'] for elem in cursor)
+
+    if operation != Transaction.CREATE:
```
Comment (sbellem) [inline bigchaindb/backend/mongodb/query.py]: Out of curiosity: Why not simply:

```python
if operation == Transaction.TRANSFER:
```

Comment (libscott) [inline bigchaindb/backend/mongodb/query.py]: Well it would have to be `if operation is None or operation == Transaction.TRANSFER`

**Comment by libscott** [comment]
Hm, accidentally pushed master and this ticket closed itself.

**Pull request merged**